### PR TITLE
Product Page

### PR DESCRIPTION
**Pull request by varun-Tad**
Created product page.Please Review

### CODE REVIEW

File: Pages/prod-listing/prod-list.css [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+.prod-land{
+    display: grid;
+    grid-template-columns: 1fr 4fr;
+    margin-top:12rem;
+    height: 100vh;
+    column-gap: 1rem;
+
+}
+
+
```
Comment (sri-rishi) [inline Pages/prod-listing/prod-list.css:114]: remove extra spacing

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Screenshot (25)
 
It's overflowing

**Comment by varun-Tad** [comment]
but it's not overflowing for me. wonder why it's overflowing for you @sri-rishi

**Comment by varun-Tad** [comment]
Can you check in another browser maybe, if it's overflowing for you because it's not overflowing for me anywhere @sri-rishi

**Comment by sri-rishi** [comment]
> but it's not overflowing for me. wonder why it's overflowing for you @sri-rishi

Then it's good to merge

File: Pages/prod-listing/prod-list.css [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+    width:33rem;
+    height:33rem;
+  }
+
+
+  .image-contain img{
+      width: 100%;
+      height: 100%;
+      border-radius: 1rem;
+  }
```
Comment (yaswanthmyneni) [inline Pages/prod-listing/prod-list.css:212]: Use object-fit property here to get clear images for carts.
Screenshot 2022-02-13 at 9 21 03 AM

File: Pages/prod-listing/prod-list.css [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+    background-color: var(--background-white);
+    height:12rem;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+    position:fixed;
+    top:0;
+    left:0;
+    right:0;
+    z-index: 11;
```
Comment (yaswanthmyneni) [inline Pages/prod-listing/prod-list.css:52]: Just a suggestion, if you want, you can do it according to the images below.
Screenshot 2022-02-13 at 9 25 52 AM
Screenshot 2022-02-13 at 9 28 21 AM
Screenshot 2022-02-13 at 9 29 30 AM

**Pull request merged**